pkg/domain/runaway.go

```diff
@@ -399,7 +399,7 @@ func (do *Domain) initResourceGroupsController(ctx context.Context, pdClient pd.
 		return nil
 	}
 
-	control, err := rmclient.NewResourceGroupController(ctx, uniqueID, pdClient, nil, rmclient.WithMaxWaitDuration(resourcegroup.MaxWaitDuration))
+	control, err := rmclient.NewResourceGroupController(ctx, uniqueID, pdClient, nil)
```

Comment:
We replace `LoadGlobalConfig` with `Get`, so need to impl `Get` in mock pd client

Reply:
updated to the latest pd client in release-7.5